components/adalo/actions/get-records/get-records.mjs

```diff
@@ -0,0 +1,20 @@
+import base from "../common/base.mjs";
+
+export default {
+  ...base,
+  key: "adalo-get-records",
+  name: "Get Records",
+  description: "Get all records from a collection. [See docs here](https://help.adalo.com/integrations/the-adalo-api/collections)",
+  version: "0.0.1",
+  type: "action",
+  async run({ $ }) {
+    const response = await this.adalo.getRecords({
+      $,
+      collectionId: this.collectionId,
+    });
```

Comment:
For actions is a good idea to support pagination as well if of course the API supports it. You can take a look at this example in [activecampaign app here](https://github.com/PipedreamHQ/pipedream/blob/master/components/activecampaign/actions/list-campaigns/list-campaigns.mjs#L19) or [amara app here](https://github.com/PipedreamHQ/pipedream/blob/master/components/amara/actions/list-teams/list-teams.mjs#L36) using generators